tests/db/update_schemas.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 set -ex
 
+python tests/store/dump_schema.py tests/resources/db/latest_schema.sql
```

Comment:
This seems to be a required step for schema migration of sqlalchemy stores. This exact step is missing in https://github.com/mlflow/mlflow/blob/master/CONTRIBUTING.rst#database-schema-changes 
- adding this step to existing DB migration script

Reply:
Nice!